src/tests/specs/ink/Bindings.spec.ts

```diff
@@ -16,15 +17,15 @@ describe("Bindings", () => {
 
     let testExternalBindingMessage = "";
 
-    story.BindExternalFunction("message", (arg: any) => {
+    story?.BindExternalFunction("message", (arg: any) => {
```

Comment:
I'm guessing we are adding these to avoid a ts warning? If so I think it may be better to type story as `Story` only, and tell ts to ignore the line where we reset it to undefined in between tests?